docs/software/media.md

```diff
@@ -1864,8 +1864,53 @@ License: [GPLv3](https://github.com/navidrome/navidrome/blob/master/LICENSE)
 
 ## Kavita
 
-T.b.d.
+Kavita is self-hosted library, for comics, manga or books. A highlight of some of its key features:
 
+- _Versatile Content Support_: Kavita ensures a comprehensive library experiencecaters by serving Manga, Webtoons, Comics (cbr, cbz, zip/rar/rar5, 7zip, raw images), and Books (epub, pdf) with a responsive reading experience.
+
+- _Efficient Organization_ with metadata richness: Kavita has a rich metadata support, enhancing user experience with effective filtering and searching. Users can efficiently organize their reading materials through Collections, Reading Lists (CBL Import), and a Want to Read feature, ensuring a well-structured library.
+
+- _Personalization and Theming_: With support for Dark mode and customizable theming, Kavita allows users to tailor their reading environment according to their preferences for a personalized and visually comfortable experience. It has a responsive interface that work great on any device (phone, tablet, desktop). Dark mode and customizable theming support
+
+
+![Kavita web interface screenshot](../assets/images/dietpi-software-media-kavita.jpg){: width="600" height="378" loading="lazy"}
+
+=== "Web interface"
+
+    The web interface uses port **2036**:
+    - URL: `http://<your.IP>:2036`
+
+=== "Setup details"
+
+    The install, config and data directory is located at:
+
+    ```
+    /mnt/dietpi_userdata/{ebooks,comics}
+    ```
+
+=== "Service control"
+
+    Kavita runs as systemd service, it can be controlled with the following commands:
+
+    ```sh
+    systemctl status kavita
+    ```
```

Comment:
```suggestion
    ```sh
    systemctl status kavita
    ```

    ```sh
    systemctl start kavita
    ```

    ```sh
    systemctl stop kavita
    ```

    ```sh
    systemctl restart kavita
    ```
```

Reply:
Shall we also direct the users to use `dietpi-services`?

Reply:
Not for starting/stopping the (single) service. It is possible but more meant for error handling/output in our scripts.

Although we could add a general info about `dietpi-services` for CPU and I/O priorities/scheduling and include/exclude for restarts during maintenance tasks to all software docs. But this would then certainly something for an import plugin, so we have a single markdown file which contains the text and link about this, and import this in every software doc.